airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -49,13 +51,20 @@ public PostgresDestination() {
   public JsonNode toJdbcConfig(JsonNode config) {
     final String schema = Optional.ofNullable(config.get("schema")).map(JsonNode::asText).orElse("public");
 
-    final StringBuilder jdbcUrl = new StringBuilder(String.format("jdbc:postgresql://%s:%s/%s",
+    List<String> additionalParameters = new ArrayList<>();
+
+    final StringBuilder jdbcUrl = new StringBuilder(String.format("jdbc:postgresql://%s:%s/%s?",
         config.get("host").asText(),
         config.get("port").asText(),
         config.get("database").asText()));
 
     if (config.get("ssl").asBoolean()) {
-      jdbcUrl.append("?ssl=true&sslmode=require");
+      additionalParameters.add("ssl=true");
+      additionalParameters.add("sslmode=require");
+    }
+
+    if (!additionalParameters.isEmpty()) {
```

Comment:
you actually don't need to check for empty. the forEach won't do anything if it is empty